### PR TITLE
Update jedi repos (fv3-jedi, soca) from develop 2023/12/12

### PR DESCRIPTION
**Pull request by climbfuji**
## Description

Temporarily change names of fv3-jedi and soca branches for update from develop. The fv3-jedi update PR was already merged, once https://github.com/JCSDA-internal/soca/pull/987 is merged I will revert the soca branch name and the only "updates" in this PR will be the dates in the comments (last updated xx/yy/zzzz).

## Issue(s) addressed

Part of ALGO-61 epic

## Dependencies

- [x] waiting on https://github.com/JCSDA-internal/soca/pull/987

## Impact

n/a

## Checklist

- [x] I have performed a self-review of my own code
- [ ] ~~I have made corresponding changes to the documentation~~
- [x] I have run the unit tests before creating the PR

### CODE REVIEW

**Review by mark-a-potts — DISMISSED** [review]

**Comment by climbfuji** [comment]
This was approved previously, pushing the reversal of the soca branch name dismissed the review. CI tests passed, so after another self-review I'll go ahead and merge.

Self-review: add back in the various `UPDATE` statements and push to the correct (old) ufs-weather-model branch, then check again.

**Pull request merged**
